pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -486,8 +486,7 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 					},
 					{
 						Alert: "KubeVirtVMIExcessiveMigrations",
-						Expr:  intstr.FromString("floor(increase(sum by (vmi) (kubevirt_migrate_vmi_succeeded_total)[1d:1m])) >= 12"),
-						For:   "1m",
+						Expr:  intstr.FromString("sum by (vmi) (max_over_time(kubevirt_migrate_vmi_succeeded_total[1d])) >= 12"),
```

Comment:
@machadovilaca As discussed, please check whether using the extra labels is needed for the succeeded migrations metric.